src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+  <small>^ Note that this particular combination requires calling `addEventListener()` with `{ passive: false }`</small>.
+
+- `document` + "keydown" event: `preventDefault()` for this combination is lethal. It renders the page largely useless, preventing keyboard scrolling, tabbing, and keyboard highlighting.
+
+- `document` + "mousedown" event: `preventDefault()` for this combination will prevent text
+  highlighting with the mouse and any other "default" action that one would invoke with a mouse
+  down.
+
+- `<input>` element + "keypress" event: `preventDefault()` for this combination will prevent characters typed by the user from reaching the input element (but don't do this; there is rarely, if ever, a valid reason for it).
+
+- `document` + "contextmenu" event: `preventDefault()` for this combination prevents the native
+  browser context menu from appearing when a user right-clicks or long-presses (or any other way in which a context menu might appear).
+
+This is not an exhaustive list by any means, but hopefully it gives you a good idea of how
+`preventDefault()` can be used.
+
+### A fun practical joke?
+
+What happens if you `stopPropagation()` _and_ `preventDefault()` in the capturing phase, starting at
+the document? Hilarity ensues! The following code snippet will render any web page just about
+completely useless:
+
+```js
+function preventEverything(e) {
+  e.preventDefault();
+  e.stopPropagation();
+  e.stopImmediatePropagation();
+}
+
+document.addEventListener('click', preventEverything, true);
+document.addEventListener('keydown', preventEverything, true);
+document.addEventListener('mousedown', preventEverything, true);
+document.addEventListener('contextmenu', preventEverything, true);
+document.addEventListener('mousewheel', preventEverything, { capture: true, passive: false });
+```
+
+I don't really know why you'd ever want to do this (except maybe to play a joke on someone), but it
+is useful to think about what's happening here, and realize why it creates the situation it does.
+
+All events originate at the document, so in this snippet, we're stopping, dead in their tracks, all
+`click`, `keydown`, `mousedown`, `contextmenu`, and `mousewheel` events from ever getting to any
+elements that might be listening for them. We also call `stopImmediatePropagation` so that any
+handlers wired up to the document after this one, will be thwarted as well.
+
+Note that `stopPropagation()` and `stopImmediatePropagation()` aren't (at least not mostly) what
+render the page useless. They simply prevent events from getting where they would otherwise go.
+
+But we also call `preventDefault()`, which you'll recall prevents the default _action_. So any
+default actions (like mousewheel scroll, keyboard scroll or highlight or tabbing, link clicking,
+context menu display, etc.) are all prevented, thus leaving the page in a fairly useless state.
+
+## Acknowledgements
```

Comment:
Is there anything else you'd like to say here - extra resources or demos? It's fine to just end, it's a reference guide, just wonddering if there's any other great events demo pages out there.

Reply:
Maybe here point to the Glitch source code, so people can explore it.